docs/content/best-practices/source-of-truth.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Source of Truth
+sidebar_position: 3
+slug: /best-practices/source-of-truth
+description: Deciding where to store the "source of truth" for authorization data
+---
+
+import {
+  ProductName,
+  ProductNameFormat,
+  RelatedSection,
+
+} from '@components/Docs';
+
+# Storing the "source of truth" for authorization data
```

Comment:
An alternative title, but also lengthy?

"When to use FGA as the 'source of truth' for authorization data"